docs/packetsampling-guide.md

```diff
@@ -0,0 +1,76 @@
+# PacketSampling User Guide
+
+Starting with Antrea v2.0, Antrea supports using PacketSampling for network diagnosis.
+It can capture specified number of packets from real traffic and upload them to a
+supported storage location. Users can create a PacketSampling CRD to trigger
+such actions on the target traffic flow.
+
+<!-- toc -->
+- [Prerequisites](#prerequisites)
+- [Start a new PacketSampling](#start-a-new-packetsampling)
+<!-- /toc -->
+
+## Prerequisites
+
+The PacketSampling feature is disabled by default. If you
+want to enable this feature, you need to set PacketSampling feature gate to true in `antrea-config`
+ConfigMap for antrea-agent.
+
+```yaml
+  antrea-agent.conf: |
+    featureGates:
+    # Enable packetsampling feature to capture real traffic packets.
+      PacketSampling: true
+```
+
+## Start a new PacketSampling
+
+When start a new packet sampling, you can provide the following information to identify
+the target flow:
+
+* Source Pod
+* Destination Pod, Service or IP address
+* Transport protocol (TCP/UDP/ICMP)
+* Transport ports
+
+You can start a new packet sampling by creating a PacketSampling CR via
+`kubectl` and a yaml file which contains the essential configuration of
+PacketSampling CRD. Following is an example of PacketSampling CR:
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: PacketSampling
+metadata:
+  name: ps-test
+spec:
+  fileServer:
+    url: sftp://127.0.0.1:22/upload # define your own ftp url here.
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: test-secret
```

Comment:
Updated this doc accoding suggestions. Also quan had a comment (https://github.com/antrea-io/antrea/pull/5821#discussion_r1569026150) to use a dedicated secret in kube-system to avoid possible permission issues.